examples/cognito_forgot_password.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+
+from aiobotocore.session import AioSession
+
+AWS_ACCESS_KEY_ID = "xxx"
```

Comment:
Access key and secret key should never be included in Python code, to reduce security risks, such as accidentally checking them into a source code repository.

Why not follow the example set in examples/sqs_queue_create.py?